src/client.ts

```diff
@@ -17,31 +17,27 @@ import { Publication } from './messages/Publication';
 import { Toast } from './messages/Toast';
 
 /**
- * Configuration options given to the client
- * from the host application.
+ * This class is the primary interface that an embedded iframe client should use to communicate with
+ * the host application.
  */
-interface ClientConfigOptions {
-  clientWindow?: Window;
-}
-
-/**
- * The Client is access point for the embedded UI's in the host application.
- */
-class Client {
+export class Client {
   private _isStarted: boolean;
```

Comment:
This is a nit pick: we typically avoid the `_` prefix in typescript because we can mark variables and functions as `private` explicitly

Reply:
I wouldn't do this in a standalone project, but the expectation is that this library may be used from within JavaScript projects, at which point all the privacy annotations are gone, so we still need some obvious indicator that folks have stepped outside of supported behavior. Unfortunately, `_` prefixing is the best we have there without breaking a bunch of TypeScript conventions.

Reply:
Gotcha. Thanks!